assets/src/design-system/components/slider/index.js

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import { useRef, useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { useKeyDownEffect } from '../keyboard';
+import { Tooltip } from '../tooltip';
+import { PLACEMENT } from '../popup';
+
+const DEFAULT_SIZE = 24;
+const rangeThumb = css`
+  appearance: none;
+  width: ${({ thumbSize }) => thumbSize}px;
+  height: ${({ thumbSize }) => thumbSize}px;
+  background-color: ${({ theme }) => theme.colors.interactiveBg.primaryNormal};
+  cursor: pointer;
+  border-radius: 50px;
+`;
+const rangeThumbHover = css`
+  background-color: ${({ theme }) => theme.colors.interactiveBg.primaryHover};
+`;
+
+const Input = styled.input.attrs({
+  type: 'range',
+})`
+  position: relative;
+  min-width: 100px;
+  cursor: pointer;
+  outline: none;
+  background: ${({ theme }) => theme.colors.bg.quaternary};
```

Comment:
@miina, I removed the gradient and just used `quaternary` back here, as this is the based track color.

Reply:
Ok, it was originally for "safety" in case the `::before` would be shifted for some reason but that shouldn't really happen.